test/PEXv2.spec.ts

```diff
@@ -126,4 +258,52 @@ describe('evaluate', () => {
     expect(proof.proofValue).toEqual('fake');
     expect(proof.verificationMethod).toEqual('did:ethr:0x8D0E24509b79AfaB3A74Be1700ebF9769796B489#key');
   });
+
+  it('should return ok if presentation definition with _const is valid', () => {
+    const pd: PresentationDefinitionV2 = getPresentationDefinitionV2_1();
+    const pex: PEXv2 = new PEXv2();
+    pd.input_descriptors![0].constraints!.fields![0].filter = {
+      type: 'string',
+      _const: 'https://yourwatchful.gov/drivers-license-schema.json',
+    };
+    const result1 = new ValidationEngine().validate([
+      {
+        bundler: new PresentationDefinitionV2VB('root'),
+        target: pd,
+      },
+    ]);
+    expect(result1).toEqual([
+      {
+        message: 'field object "filter" property must be valid json schema',
+        status: 'error',
+        tag: 'presentation_definition.input_descriptor[0].constraints.fields[0]',
+      },
+    ]);
+    const newPd = JsonPathUtils.changePropertyNameRecursively(
+      pd as InternalPresentationDefinitionV2,
+      '_const',
+      'const'
+    );
+    const result2 = pex.validateDefinition(newPd);
```

Comment:
Isn't pex.ValidationDefinition not already doing the name change recursively, given it is a main entry point for users?